masonry/src/event.rs

```diff
@@ -30,14 +30,34 @@ pub enum WindowEvent {
     RebuildAccessTree,
 }
 
+/// An indicator of which pointer button was pressed.
+#[derive(PartialEq, Eq, Clone, Copy, Debug, Hash)]
+#[repr(u8)]
+pub enum PointerButton {
+    /// No mouse button.
+    None,
+    /// Primary button, commonly the left mouse button, touch contact, pen contact.
+    Primary,
+    /// Secondary button, commonly the right mouse button, pen barrel button.
+    Secondary,
+    /// Auxiliary button, commonly the middle mouse button.
+    Auxiliary,
+    /// X1 (back) Mouse.
+    X1,
+    /// X2 (forward) Mouse.
+    X2,
+    /// Other mouse button. This isn't fleshed out yet.
```

Comment:
According to https://www.w3.org/TR/pointerevents3/#the-button-property, there's also the Pen Eraser button? I guess if winit doesn't expose that, it's not really that important

Reply:
Should we file a bug to follow up on this?

Reply:
Probably...